### DISCUSSION TITLE
Training a model to forecast a TS for several subjects.

### DISCUSSION THREAD

**Discussion by darleybarreto** [ourownstory/neural_prophet#388] [Q&A - get help using NeuralProphet] [answered]
Hi folks, first I would like to say great job! For what I understood by looking at the examples, I can only train a model for each TS. So if there's several TS, one for each subject of the dataset, I would need to train one model for each one. Is that correct?

Thanks in advance!

**Comment by ourownstory** [comment, marked as answer]
Hi @darleybarreto 

Thank you. Yes, that is correct.
You can have multiple input series (exogenous variables/regressors), but only one target series.

**Reply by darleybarreto** [reply]
Would it be possible to have features and target in different timestamps? For instance, the features are collected daily and the targets could be monthly?

**Reply by wangzhongqing** [reply]
Excuse me, can you save the model file to local？

**Reply by ourownstory** [reply]
@darleybarreto That is a great suggestion! Would you like to support this project by implementing it? Added your suggestion as #409
